sources/src/main/java/com/google/solutions/jitaccess/web/IapRequestFilter.java

```diff
@@ -53,6 +57,7 @@ public class IapRequestFilter implements ContainerRequestFilter {
 
   private static final String IAP_ISSUER_URL = "https://cloud.google.com/iap";
   private static final String IAP_ASSERTION_HEADER = "x-goog-iap-jwt-assertion";
+  private static final String IAP_CLOUD_RUN_SKIP_VERIFICATOIN_HEADER_VALUE = "SKIP_IAP_VERIFICATION";
```

Comment:
Instead of using a magic value, I'd prefer to make this an extra setting/environment variable